docs/root/intro/version_history.rst

```diff
@@ -37,6 +37,7 @@ Features:
 - api: add option to control whether Envoy should drain connections after a soft DNS refresh completes. (:issue:`#2225 <2225>`, :issue:`#2242 <2242>`)
 - api: add option to disable the gzip decompressor. (:issue: `#2321 <2321>`) (:issue: `#2349 <2349>`)
 - api: add option to enable the brotli decompressor. (:issue `#2342 <2342>`) (:issue: `#2349 <2349>`)
+- api: add option to enable socket tagging. (:issue `#1512 <1521>`)
```

Comment:
Oh you went whole hog and added an explicit API.
I think we should call out what socket tagging is somewhere as well.  I'm not sure what the Envoy-Mobile docs requirements are, but in Envoy we'd usually explain in the proto file and link autogenerated docs.  not sure if E-M does the autogen docs but best case you can grep for a prior example, worst case you can doc up the proto and hard link to it (and Lyfties can correct i that's wrong)

Reply:
Sorry! As you uncovered below, I misunderstood the PR I was copying from. I'm happy to make this *not* part of the API if that would be better. In any case, I've added some comments to the proto to attempt to document this API.

But very happy to do something simpler!

Reply:
cool.  and either repo link to the proto or reflink the docs here?

Reply:
Hm. This sounds like a good idea, but I don't quite see how to do that based on existing entries in this file. Do you have an example I can start with?

Reply:
I don't know if there are exampes.  @jpsim thoughts here?

Reply:
I think that `[starting_envoy.rst](https://github.com/envoyproxy/envoy-mobile/blob/main/docs/root/api/starting_envoy.rst)` file may be a good home for what you are planning to add.

Reply:
@RyanTheOptimist did you have a chance to look at the file I referenced above?